### ISSUE TITLE
Autonomiser les utilisateurs vis à vis de la création de sites et de mailles

### ISSUE THREAD

**Issue by jpm-cbna** [PnX-SI/gn_module_suivi_flore_territoire#68]
Actuellement, la création de sites et de mailles passe par l'utilisation des scripts Bash mis à disposition dans le module.
L'utilisation de ces scripts nécessite des droits permettant d'accéder au serveur hébergeant le module.
Afin de réduire ce goulot d'étranglement,  il serait intéressant de trouver une solution pour le faire via l'interface afin d'autonomiser les utilisateurs.

Toutefois, il est nécessaire de garder en tête qu'il faut éviter une trop grande dispersion au niveau de l'ajout des sites et des mailles. Cela nécessite donc de définir clairement des règles précises concernant l'ajout des sites et mailles.

Ce type d'ajout pourrait être restreint de plusieurs façons:
 - autoriser seulement  a certains utilisateurs par des droits spécifiques autorisant l'ajout des sites et mailles
 - autoriser à tous les utilisateurs ayant des droits d'écriture mais avec une modération (envoie d'email à des modérateurs) avant publication définitive.

N'hésitez pas à commentez ce ticket et à proposer vos solutions vis à vis de ce problème !

**Comment by camillemonchicourt** [comment]
Oui je privilégierai la solution 1.
A voir si pour les mailles, une fois qu'on toutes celles du territoire dans la BDD, seul l'ajout de sites est nécessaire, les mailles étant intersectées par les sites automatiquement de mémoire.
Du coup uniquement une interface pour saisir des sites réservée aux administrateurs/référents ou une fonctionnalité d'import, me paraissent suffisant ?

**Comment by jpm-cbna** [comment]
Oui, j'avais tester l'ajout des mailles en question par département. Malheureusement, leur taille est vraiment trop petite vis à vis du territoire concerné. Du coup, cela représente une énorme quantité de mailles dont très très peu servent réellement. 
Cela alourdie la base de données et la table `l_areas` et du coup impacte certains triggers sur la synthèse.
Bref, c'est difficilement utilisable.

C'est peut être malgré tout à tenter à nouveau pour voir l'impact car depuis les triggers se sont améliorer et j'ai amélioré mes scripts d'intégration des zones géographiques...
